test/test-connector-swagger2.js

```diff
@@ -62,34 +62,33 @@ describe('OpenAPI connector for Swagger 2.0', function() {
   });
 
   describe('models', function() {
-    describe('models without remotingEnabled', function() {
-      let ds, PetService, petId;
-      before(async () => {
-        ds = await createDataSource('test/fixtures/2.0/petstore.json');
-        PetService = ds.createModel('PetService', {});
-        const data = await PetService.findPetsByStatus({status: 'available'});
-        should(data.body).be.Array();
-        should(data.body.length).be.above(0);
-        petId = data.body[0].id;
-      });
+    let ds, PetService, petId;
+    before('connect to pet store', async () => {
+      ds = await createDataSource('test/fixtures/2.0/petstore.json');
+      PetService = ds.createModel('PetService', {});
+      const data = await PetService.findPetsByStatus({status: 'available'});
+      should(data.body).be.Array();
+      should(data.body.length).be.above(0);
+      petId = data.body[0].id;
+      console.log('Pet id: %s', petId);
```

Comment:
nitpick: do we have to print the id?

Reply:
The pet store dataset is not stable. I'm printing out the pet id for troubleshooting.

Reply:
Now I use the last id for testing. It seems to be more reliable.